docs/_source/tutorials/notebooks/labelling-tokenclassification-basics.ipynb

```diff
@@ -198,6 +187,36 @@
     "As a first step, we want to get the list of the users that will be annotating our dataset."
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import argilla as rg\n",
+    "\n",
+    "rg.init(\n",
+    "    api_url=\"...\",\n",
+    "    api_key=\"...\"\n",
+    ")\n",
+    "\n",
+    "# get the list of users\n",
+    "# optional: filter users to get only those with annotator role\n",
+    "users = [u for u in rg.User.list() if u.role == \"annotator\"]"
```

Comment:
The following suggestion addresses these issues:
- Listing users is limited to the role `owner`, but perhaps an `admin` might be doing the assignments. 
- We want to make sure that we only make assignments to users that have access to the dataset.
```suggestion
    "# get workspace where the dataset is (or will be) located",
    "ws = rg.Workspace.from_name("my_workspace")",
    "# get the list of users with access to the workspace\n",
    "# make sure that all users that will work on the dataset have access to the workspace\n",
    "# optional: filter users to get only those with annotator role\n",
    "users = [u for u in ws.users if u.role == \"annotator\"]"
```